src/components/createPostPopup/surveyPost.tsx

```diff
@@ -1,111 +1,143 @@
+            />
           </Col>
-          <Col sm="3">
-            <Form.Control as="textarea" placeholder="Option 2 (mandatory)" />
+          <Col>
+            <Form.Control
+              id="option2"
+              name="option2"
+              as="textarea"
+              required
+              minLength={1}
+              placeholder="option 2 (mandatory)"
+            />
           </Col>
         </Row>
-        <br></br>
-        <Row>
+        <Row className="my-2">
           <Col sm="3" />
-          <Col sm="3">
-            <Form.Control as="textarea" placeholder="Option 3" />
+          <Col>
+            <Form.Control
+              id="option3"
+              name="option3"
+              as="textarea"
+              placeholder="option 3"
+            />
           </Col>
-          <Col sm="3">
-            <Form.Control as="textarea" placeholder="Option 4" />
+          <Col>
+            <Form.Control
+              id="option4"
+              name="option4"
+              as="textarea"
+              placeholder="option 4"
+            />
           </Col>
         </Row>
-        <br></br>
-        <Row>
+        <Row className="my-2">
           <Col sm="3">
-            <p>Choose Tags:</p>
+            <p>Endtime</p>
           </Col>
           <Col>
-            <Form.Select size="lg">
-              <option>Choose tags which best describe your post</option>
-              <option>Large select</option>
-              <option>Test</option>
+            <Form.Select id="endtime" name="endtime" required size="lg">
+              <option>1 hour</option>
+              <option>3 hours</option>
+              <option>1 day</option>
             </Form.Select>
           </Col>
         </Row>
-        <br></br>
-        <Row>
+        <Row className="my-2">
           <Col sm="3">
-            <p>Choose Endtime:</p>
+            <p>Tags</p>
           </Col>
           <Col>
-            <Form.Select size="lg">
-              <option>Choose when the survey ends</option>
-              <option>1 hour</option>
-              <option>2 hours</option>
-              <option>1 day</option>
-            </Form.Select>
+            <Form.Control
+              id="tags"
+              name="tags"
+              as="textarea"
+              placeholder="#question #opinion"
+            />
           </Col>
         </Row>
-        <br></br>
         <Row>
-          <Form.Check
-            type="switch"
-            id="custom-switch"
-            label="Check this switch"
-          />
+          <Col>
+            <Form.Check
+              type="switch"
+              id="readOnly"
+              name="readOnly"
+              label="Disable comments"
+            />
+          </Col>
         </Row>
       </Form>
 
       <style jsx>{`
+        .test {
+          font-size: 12px;
+          color: red;
+        }
+
         p {
-          color: white;
-          font-size: 26px;
+          font-size: 23px;
```

Comment:
absolute größten (`px`) sind sehr schlecht
der bootstrappigere weg wäre das so oder so oben zu setzen, die textgröße, in `className`:
https://getbootstrap.com/docs/5.3/utilities/text/#font-size